--- original

+++ new

@@ -26,7 +26,7 @@

 | last_aired         = {{end date|2003|3|4|df=y}} 
 }}
 
-{{Use dmy dates|date=January 2014}}
+{{Convert to Episode table}}{{Use dmy dates|date=January 2014}}
 {{Use British English|date=January 2014}}
 
 '''''Buried''''' is a British [[television]] prison drama, produced by [[World Productions]] for [[Channel 4]] and originally broadcast between 14 January and 4 March 2003.<ref name=":0">{{Cite book|title=State of play : contemporary "high-end" TV drama|last=Nelson|first=Robin|date=2007|publisher=Manchester University Press|isbn=9781847791825|location=Manchester|pages=177–183|oclc=652139398}}</ref> The eight-part series starred [[Lennie James]], [[Stephen Walters]], [[Connor McIntyre]], [[Jane Hazlegrove]], and [[Neil Bell (actor)|Neil Bell]] in key roles.