--- original

+++ new

@@ -1,6 +1,6 @@

 {{Short description|Drama TV series (1961-62)}}
 {{more citations needed|date=December 2010}}
-{{Use mdy dates|date=March 2025}}
+{{Convert to Episode table}}{{Use mdy dates|date=March 2025}}
 {{Infobox television
 | image                = Marilyn Maxwell Tuesday Weld Gary Lockwood Bus Stop 1961.JPG
 | caption              = [[Marilyn Maxwell]], [[Tuesday Weld]] and [[Gary Lockwood]] in "Cherie" (1961)